outlook/calendar/main.go

```diff
@@ -70,6 +70,11 @@ func main() {
 		if id := os.Getenv("CALENDAR_ID"); id != "" {
 			info.ID = id
 			info.Owner = graph.OwnerType(os.Getenv("OWNER_TYPE"))
+
+			if info.Owner == "" {
+				fmt.Println("Owner type is required")
+				os.Exit(1)
+			}
```

Comment:
The LLM was often forgetting to set this parameter. I improved the prompting in the parameter description, so it shouldn't forget as often, but if it does, we now return an error so that it retries.